iex-cloud/src/endpoint/crypto.ts

```diff
@@ -0,0 +1,39 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+
+export const NAME = 'crypto'
+
+const customParams = {
+  base: ['base', 'from', 'coin', 'asset', 'symbol'],
+  quote: ['quote', 'to', 'market'],
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const base = validator.validated.data.base
+  const quote = validator.validated.data.quote
+  const url = `${base.toUpperCase()}${quote.toUpperCase()}/quote`
+
+  const params = {
+    token: config.apiKey,
+  }
+
+  const reqConfig = {
+    ...config.api,
+    params,
+    baseURL: 'https://cloud.iexapis.com/stable/crypto',
```

Comment:
This could come from `config`, and then you add the endpoint? Same for `stock`. 
`${config.baseURL}/crypto`, `${config.baseURL}/stock`

Reply:
I was following the format of the example adapter. Is this something we should change there as well?

Reply:
I'm not very familiar with the example adapter, I'm sure @krebernisak can bring some light here 😄